apps/mosaico/src/Main.purs

```diff
@@ -350,6 +354,9 @@ frontpageUpdated env { params: { category } } = do
 assets :: { params :: { path :: List String } } -> Aff (Either Failure File)
 assets { params: { path } } = Handlers.directory "dist/assets" path
 
+adsTxt :: forall r. { | r} -> Aff File
+adsTxt = Handlers.file "dist/assets/ads.txt"
```

Comment:
Don't know if we need it under `dist`. Do we want to serve it from `hbl.fi/assets/ads.txt` too? 🤔

Reply:
Oh, nevermind, I see what you did there! Does this always read our file system on request? I guess it'll be cached in nginx anyway so maybe not a big deal.

Reply:
It does always read the file system, yes (unless nginx caches it). Google tries to update the file once every 24 hours according to their docs, so I imagine it won't create a huge amount of extra load.